src/util/promise.js

```diff
@@ -13,6 +13,10 @@ export let mapValuesAsync = _.curry((f, d) => promisedProps(_.mapValues(f, d)))
 export let flowAsyncF = (...fns) => (...x) =>
   fns.slice(1).reduce((v, f) => v.then(f), Promise.resolve(fns[0](...x)))
 export let flowAsync = (...args) => {
```

Comment:
Comment explaining what `...args` can be might be helpful. Same with `flowAsyncF`.

Reply:
it's async version of lodah `flow`, which was planned for futil - I'm not sure why duti decided to modify these files  🤔

Until they make it into futil, it's probably worth a least a comment explaining that 👍